src/main/java/com/java/aop/HomeAspect.java

```diff
@@ -9,8 +9,8 @@
 
 @Component
 @Aspect
-public class HomeAscpect {
-	public static Logger logger = Logger.getLogger(HomeAscpect.class.getName());
+public class HomeAspect {
+	public static Logger logger = Logger.getLogger(HomeAspect.class.getName());
```

Comment:
`logger` 는 대부분 `private static final` 로 선언해서
클래스 별로 가지고, 해당 클래스에서만 사용하는게 보통

```suggestion
	public static Logger logger = Logger.getLogger(HomeAspect.class);
```

그리고 getName 까지 없어도 동작하지 않나?